traffic_ops/install/bin/postinstall.test.sh

```diff
@@ -0,0 +1,424 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e;
+
+if [[ ! -x /usr/bin/python3 ]]; then
+	echo "Python 3.6+ is required to run - or test - postinstall.py" >&2;
+	exit 1;
+fi
+
+ROOT_DIR="$(mktemp -d)";
```

Comment:
`ROOT_DIR` should not be all caps because it is not an environment variable.

Reply:
says who?

Reply:
From [Google's shell style guide](https://github.com/google/styleguide/blob/gh-pages/shellguide.md#constants-and-environment-variable-names):

>### Constants and Environment Variable Names
>
>All caps, separated with underscores, declared at the top of the file.
>
>Constants and anything exported to the environment should be
>capitalized.
>
>```shell
># Constant
>readonly PATH_TO_FILES='/some/path'
>
># Both constant and environment
>declare -xr ORACLE_SID='PROD'
>```

In other words, if you want ROOT_DIR to be a constant, it should
* Be declared using `readonly` or `declare -r`
* Be declared at the top of the file instead of inline

Reply:
Granted, declaring inline vs at the top is more subjective.

Reply:
Google ain't the boss of me. But I don't have a problem marking things that should be readonly with `readonly`.

Reply:
I resolved all the other conversations about the same thing so we can just discuss it here